server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java

```diff
@@ -96,14 +96,13 @@ public static Book getBook(DatabaseClient dbClient, String bookId) {
               Key.of(bookId),
               Arrays.asList("title", "autho", "isbn"));
```

Comment:
Spelling: This slipped through my last PR so this is on me, but this should be "author."

Reply:
Fix pushed.